_includes/banner-news.html

```diff
@@ -0,0 +1,7 @@
+<!-- News banner BEGIN -->
+<div class="site-banner-news" >
+    <p><strong>USWDS 2.0 is coming soon.</strong>
+    See what’s new in Version 2.</p>
+    <a href="https://v2.designsystem.digital.gov" onclick="ga('send', 'event', 'Viewed 2.0 preview', 'Clicked to view 2.0 version from inside site');"><button class="usa-button usa-button-secondary-inverse">Try it now</button></a>
```

Comment:
Instead of wrapping a `button` in an `a`, just give the `a` element the button classes: `usa-button usa-button-secondary-inverse`
```suggestion
    <a href="https://v2.designsystem.digital.gov" onclick="ga('send', 'event', 'Viewed 2.0 preview', 'Clicked to view 2.0 version from inside site');" class="usa-button usa-button-secondary-inverse">Try it now</a>
```